website/project/metadata/aea-prereg-2.json

```diff
@@ -0,0 +1,480 @@
+{
+    "name": "AEA_Preregistration",
+    "version": 2,
+    "description": 
+    "pages": [{
+        "id": "page1",
+        "title": "",
+        "contents": [{
+            "id": "general",
+            "type": "section",
+            "contents": [{
+                "required": true,
+                "type": "textfield",
+                "id": "title",
+                "label": "Trial Title",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_number",
+                "label": "AEA RCT Registration Number",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_date",
+                "label": "Registration Date",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "updated",
+                "label": "Last Updated",
+                "caption": "caption"
+            }],
+            "title": "General Information"
+        }, {
+            "type": "section",
+            "canRepeat": true,
+            "id": "location",
+            "contents": [{
+                "type": "select",
+                "required": true,
+                "options": [
```

Comment:
I'm trying to get a better grasp on what this PR encompasses; can you clarify? I know this PR has a pretty tangled history.

It seems that the only way for an inaccurate file to be out of scope is if the file is not used in this PR at all. In which case I'm unclear as to why it is included here (rather than in a follow-on PR).

We will need a way to track such followup issues for later when they do become relevant. I want this to move fast, but just letting things slip through review and be forgotten seems like an invitation to trouble. I've pinged David on this separately, to take advantage of his perspective.